documentation/documentation.css

```diff
@@ -0,0 +1,149 @@
+.navigation {
+    width: 100%;
+    height: 10vh;    
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
+    justify-content: flex-end;
+    padding: 1rem 1.5rem 0;
+    z-index: 1;
+    top: 0;
+}
+
+.documentation-container > .navigation{
+    height: 12vh;
+    background-color: var(--hover-grey);
+
+}
+
+
+
+.navigation .link {
+    color: var(--black);
+    font-weight: bold;
+}
+
+.nav-pills{
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+}
+
+.navigation .nav-brand {
+    font-family: 'Comforter', cursive;
+    font-weight: bolder;
+    letter-spacing: 0.1rem;
+    font-size: 2.8rem;
+    flex-grow: 1;
+}
+
+.navigation .link-active {
+    font-weight: bolder;
+    background-color: var(--grey);
+    padding: 0.7rem;
+    border-radius: 0.3rem;
+}
+
+.link {
+    box-sizing: border-box;
+    text-decoration: none;
+    padding: 0 1rem;   
+}
+
+.list-non-bullet {
+    list-style: none;
+}
+
+.list-item-inline {
+    display: inline;
+}
+
+.documentation-container{
+    display: flex;
+    flex-direction: column;
+    gap: 20px;
+}
+
+.header{
+    display: flex;
+    flex-direction: row;
+    color: var(--light-raisin-black);
+}  
+
+.hero-heading{
+    font-weight: bold;
+    padding: 2.5rem 0;
+}
+
+.side_bar{    
+    width: 16rem; 
+    position: sticky;
+    overflow-y: auto;
+    top: 0;
+    background-color: white;
+    text-align: center;
+    height: calc(100vh - 2rem);    
+}
```

Comment:
A bit of sidebar is visible on mobile screens, better to use a media query for smaller screens and make the  `.side_bar` to `display:none`